lstchain/calib/camera/flatfield.py

```diff
@@ -278,7 +278,7 @@ def calculate_time_results(
                                              pixel_median > self.time_cut_outliers[1])
 
         return {
-            'sample_time': (trigger_time + time_start).value / 2 *u.s,
+            'sample_time': (time_start +(trigger_time - time_start) / 2).value *u.s,
```

Comment:
Please do not use `.value` here. What do you want to store here? If you want the time, just leave it as a time and the ctapipe table writer will convert it when writing / reading.

If you need a specific format for some reason, access that (e.g. unix or unix_tai) but not `.value`.